lib/liquid/block_body.rb

```diff
@@ -113,7 +115,7 @@ def blank?
     end
 
     def render(context)
-      render_to_output_buffer(context, '')
+      render_to_output_buffer(context, +'')
```

Comment:
I'm not sure if this is the best approach. This means frozen strings will fail if passed in to `render_to_output_buffer`. The other option is to allow frozen strings to be passed in and `render_to_output_buffer` does the unfreezing

Reply:
I guess this is equivalent to what it is now.

Reply:
Yes. The API is clear. This expects an unfrozen string. If a frozen string is sent anyways, then a `FrozenError` exception will be called with the caller at fault.